Examples/MessengerClone/SceneDelegate.swift

```diff
@@ -11,14 +11,57 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     var window: UIWindow?
 
     func scene(_ scene: UIScene, willConnectTo session: UISceneSession, options connectionOptions: UIScene.ConnectionOptions) {
-        let chatChannelList = MessengerChatChannelList()
-
         // Use a UIHostingController as window root view controller.
         if let windowScene = scene as? UIWindowScene {
             let window = UIWindow(windowScene: windowScene)
-            window.rootViewController = UIHostingController(rootView: chatChannelList)
+            window.rootViewController = SplashViewController { [unowned window] in
+                window.rootViewController = UIHostingController(rootView: MessengerChatChannelList())
+            }
             self.window = window
             window.makeKeyAndVisible()
         }
     }
 }
+
+private final class SplashViewController: UIViewController {
```

Comment:
We could add the same `SplashViewController` to multiple targets, instead of copying the code.

Reply:
What is the best place to have shared components like this one?

Reply:
We can add a `Shared` folder under `Examples` and put the common code there.